django/test/testcases.py

```diff
@@ -313,10 +317,29 @@ def assertRedirects(self, response, expected_url, status_code=302,
                     % (path, redirect_response.status_code, target_status_code)
                 )
 
-        self.assertEqual(
-            url, expected_url,
-            msg_prefix + "Response redirected to '%s', expected '%s'" % (url, expected_url)
-        )
+        message = msg_prefix + "Response redirected to '%s', expected '%s'" % (url, expected_url)
+        self.assertURLEqual(url, expected_url, message)
```

Comment:
Can we maintain the old formatting (using the hanging indent) to avoid the temporary variable?